src/Lesson5/HomeTask5Class1.java

```diff
@@ -0,0 +1,30 @@
+package Lesson5;
+
+import Lesson5.Class.HomeTask5Class2;
+
+//Create 2 classes with the same name, but put them in different packages. For example, com.domainname.modulename.
+//Create a third class with the main method.
+//In the main method create two different objects of your classes specifying the full name of the classes (including packages).
```

Comment:
Re-read task and do correct changes in your code + don't forget about naming convention.